### PR TITLE
Add license files

### PR DESCRIPTION
**Pull request by bobleesj**

### CODE REVIEW

**Comment by bobleesj** [comment]
Will need to update toml as well. One moment.

File: pyproject.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ classifiers = [
         'Environment :: Console',
         'Intended Audience :: Developers',
         'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: BSD License',
+        'License :: Other/Proprietary License',
```
Comment (bobleesj) [inline pyproject.toml:23]: https://pypi.org/classifiers/

Using the convention from above.

**Review by bobleesj — COMMENTED** [review]
Ready for review @sbillinge

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,137 @@
+OPEN SOURCE LICENSE AGREEMENT
```
Comment (sbillinge) [inline LICENSE.txt:1]: We would like this to be LICENSE.rst, but rst format but with the text from LICENSE.txt

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+=============================
+
+Copyright (c) 2009-2011, University of Tennessee
```
Comment (sbillinge) [inline LICENSE.txt:4]: we need to update the copyright so that 2019-2024 is columbia again.

File: LICENSE_DANSE.txt [hunk 4]
```diff
@@ -0,0 +1,41 @@
+This program is part of the DiffPy and DANSE open-source projects
```
Comment (sbillinge) [inline LICENSE_DANSE.txt:1]: This can stay as it is.  Probably not worth changing to rst.

File: pyproject.toml (hunk 1 above)
Comment (sbillinge) [reply pyproject.toml:23]: this is not proprietary.  Is there one that says something like free for use with restrictions or sthg that actually matches what hte license says?

**Review by sbillinge — COMMENTED** [review]
please see comments.

File: pyproject.toml (hunk 1 above)
Comment (bobleesj) [reply pyproject.toml:23]: `License :: Free To Use But Restricted`

How about this?

**Comment by bobleesj** [comment]
Addressed 

- License classifier on pytoml
- Added years to copyright to LICNESE.rst
- changed LICENSE.txt to LICENSE.rst

Ready for review

File: LICENSE.rst [hunk 5]
```diff
@@ -1,30 +1,141 @@
...
+
+Copyright (c) 2008-2012, The Trustees of Columbia University in the City of New York
+
+Copyright (c) 2009-2011, University of Tennessee
+
+Copyright (c) 2014, Australian Synchrotron Research Program Inc., ("ASRP")
+
+Copyright (c) 2014-2019, Brookhaven Science Associates, Brookhaven National Laboratory
+
+Copyright (c) 2019-2024, The Trustees of Columbia University in the City of New York.
```
Comment (bobleesj) [inline LICENSE.rst:18]: Added 2019-2024

File: pyproject.toml [hunk 6]
```diff
@@ -20,7 +20,7 @@ classifiers = [
         'Environment :: Console',
         'Intended Audience :: Developers',
         'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: BSD License',
+        'License :: Free To Use But Restricted',
```
Comment (bobleesj) [inline pyproject.toml:23]: Available!

File: pyproject.toml (hunk 1 above)
Comment (bobleesj) [reply pyproject.toml]: Please see the new update.

`License :: Free To Use But Restricted` is used.

File: LICENSE.txt (hunk 3 above)
Comment (bobleesj) [reply LICENSE.txt]: done

File: LICENSE.txt (hunk 2 above)
Comment (bobleesj) [reply LICENSE.txt]: done

File: LICENSE_DANSE.txt (hunk 4 above)
Comment (bobleesj) [reply LICENSE_DANSE.txt:1]: done

**Pull request merged**